shell.nix

```diff
@@ -24,7 +13,7 @@ pkgs.mkShell {
     jq
   ];
 
-  RUST_SRC_PATH = "${pkgs.rust-src}/lib/rustlib/src/rust/library";
+  # RUST_SRC_PATH = "${pkgs.rust-src}/lib/rustlib/src/rust/library";
```

Comment:
I have something like that in in my flakes so that rust-analyzer can find the correct version (currently used) of standard library source code.

Reply:
But in that case it it would need to be commented in, right? Idk if the env var is _needed_, I assume that @wiredhikari checked that, but at least commented put it seemed to make little sense.

Reply:
Yes, I'm just saying that you might want it actually, maybe?

My understanding is that if it's isn't set here, then rust-analyzer will rely on system settings. So if someone has a `RUST_SRC_PATH` set outside of `nix develop` and inherits it, things will mostly work, but if not, they won't. Also if the version is different there might be some little confusion (but not much, because stdlib doesn't change a lot).

Reply:
That line was valid when we were using `fenix` and wont work without it. Adding a relevent path or other workaround to add support for rust-analyzer.